scripts/2.mutation-classifier.py

```diff
@@ -326,19 +326,45 @@ def get_threshold_metrics(y_true, y_pred):
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
+# ### Top coefficients for covariates model
+
 # In[21]:
 
-# Top standardized coefficients
-(coef_df
-    .query("feature_set == 'full'")
-    .head(10)
-)
+coef_df.query("feature_set == 'covariates'").head(10)
 
 
-# ## Investigate the predictions
+# ### Top coefficients for full model
 
 # In[22]:
 
+coef_df.query("feature_set == 'full'").head(10)
+
+
+# ### Top coefficients for individual _genes_ for expressions model
```

Comment:
I agree that we would be more interested in the coefficients for the `full` model. I think that the `expression` model really just serves as a benchmark for comparison of AUROC.